src/building/suggested.md

```diff
@@ -70,6 +71,17 @@ If you're running `coc.nvim`, you can use `:CocLocalConfig` to create a
 `editor.formatOnSave: true,` with
 `"coc.preferences.formatOnSaveFiletypes": ["rust"],`.
 
+> NOTE: The configuration once suggested disabling build scripts, but after
+> the version `2022-04-14` this was fixed by introducing an option for customizing
+> the command rust-analyzer uses to run build scripts. Please ensure that `rust-analyzer`
+> is up to date before enabling these options. If you do not wish to enable
+> `rust-analyzer.cargo.useRustcWrapperForBuildScripts`, the
```

Comment:
Above the option is called `rust-analyzer.cargo.buildScripts.useRustcWrapper`, but here it is called by a different name?

Reply:
It got renamed, `rust-analyzer.cargo.buildScripts.useRustcWrapper` is the new name and `rust-analyzer.cargo.useRustcWrapperForBuildScripts` is the old name that would still work.

Reply:
I think this sentence should be left out anyway.